docs/analyzer/user_guide.md

```diff
@@ -717,6 +717,11 @@ analyzer arguments:
                         clang-tidy checkers. The file can be dumped by
                         'CodeChecker analyzers --dump-config clang-tidy'
                         command.
+  --checker-config [CHECKER_CONFIG [CHECKER_CONFIG ...]]
```

Comment:
Please extend the user guide for the check command too: https://github.com/Ericsson/codechecker/blob/master/docs/analyzer/user_guide.md#check-